src/katsdpcontroller/master_controller.py

```diff
@@ -254,10 +254,9 @@ def __init__(self, args: argparse.Namespace, server: aiokatcp.DeviceServer,
         self._image_resolver_factory = image_resolver_factory
         self._rewrite_gui_urls = rewrite_gui_urls
 
-    @abstractmethod
```

Comment:
mypy was complaining that subclasses were using `super()` to call the abstract method. I think it reckons abstract methods should be pure virtual.